client/src/components/EventItem.js

```diff
@@ -35,6 +40,29 @@ function EventItem(props) {
     props.description
   );
 
+  useEffect(() => {
+    const data = {
+      userId: props.user_id,
+      eventId: props.id
+    };
+
+    fetch('/api/test', {
```

Comment:
The '/api/test/' is not a good route. The name of the route should have something to do with what it is doing.

Reply:
fixed